src/scorecard.js

```diff
@@ -0,0 +1,78 @@
+function strict(){
+  'use strict';
+}
+
+function Scorecard(){
+  this.total = 0
+  this.frameTotal = 0
+  this.frame = 1
+  this.roll  = 1
+  this.MAX_FRAME_TOTAL = 10
+}
+
+Scorecard.prototype.getCurrentScore = function () {
+  return this.total;
+}
+
+Scorecard.prototype.getFrameScore = function () {
+  return this.frameTotal;
+}
+
+Scorecard.prototype.getCurrentFrame = function () {
+  return this.frame;
+}
+
+Scorecard.prototype.getCurrentRoll = function () {
+  return this.roll;
+}
+
+Scorecard.prototype.updateScore = function (number) {
```

Comment:
you are using the argument name `number` in a lot of your functions, but it's not consistent with the domain language - you don't often use the term `number` when bowling.